promscale/tobs/tobs.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Install the observability stack (tobs) for Kubernetes
+excerpt: Install tobs
+product: promscale
+keywords: [analytics, Kubernetes]
+tags: [install, k8s]
+related_pages:
+  - /promscale/:currentVersion:/recommendations/resource-recomm
+  - /promscale/:currentVersion:/send-data/
+---
+
+# Install the observability stack (tobs) for Kubernetes
+The observability stack (tobs), is a pre-packaged distribution of observability
+tools and components which can be installed on any existing Kubernetes
+cluster. It includes many of the most popular open-source observability tools
+including Prometheus, Grafana, TimescaleDB, and [other components][stack-components].
+Together, these provide a unified solution to observe deployments in Kubernetes
+cluster and identify any potential problems using both metrics and traces. You can use Helm
+charts to configure and update [tobs][tobs] deployments. 
+
+ 
+## Before you begin
+*  Understand the [resource requirements][resource-requirements] for your cluster.
+*  Ensure that you have a [Kubernetes Cluster][kubernetes-cluster]. 
+*  Install [kubectl][kubectl] in your local environment and [connect to your cluster][connect-to-cluster].
+*  Install [Helm 3][helm] in your local machine.
+* Install a certificate manager for your cluster:
+   ```bash
+   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml
```

Comment:
The latest is already `v1.9.1`. There is no point in specifying the instruction and mentioning the highlight to check the latest and updated instructions in cert-manager docs. We are offering two different things for the same action. 

I would prefer to just reference to cert-manager docs for simplicity and maintainability.

Reply:
As I mentioned in my previous comment this is to avoid churn.

Reply:
But this doesn't make sense we providing instruction and highlighting for updated instruction go to cert-manager docs. We can either do 1/2 below two options:

1. Specify the latest cert-manager instructions here. (we should update this in a timely fashion as the new cert-manager release is landed).
2. Reference to cert-manager install instruction. 

Having it as we have now in PR at a point makes cert-manager instruction incompatible with Tobs and the user can understand this after installation failure of tobs deployment.

Reply:
added the <LATEST_VERSION> variable to the command.

Reply:
No, that doesn't help, now the user has to go to cert-manager docs. What's the point of specifying the instruction on our end?

Rajie, I understand your motivation to reduce the churn you wanted to specify on our end. But now the user has to open cert-manager docs pick the latest version from there and apply it in the command we offered. How about the user directly copy pasting the instruction in cert-manager?

Reply:
removed the command.